app/src/main/java/ghcharactertracker/com/ghcharactertracker/CharClass.java

```diff
@@ -0,0 +1,67 @@
+package ghcharactertracker.com.ghcharactertracker;
+
+/**
+ * Created by jesse.mailhot on 3/6/2018.
+ */
+
+public class CharClass {
+    ClassName name;
+    int[] healthVals;
+    int[] lvlUpVals;
+
+    public CharClass(ClassName name) {
+        this.name = name;
+        healthVals = CharClass.getHealthValues(this.name);
+        lvlUpVals = CharClass.getLvlUpValsValues(this.name);
+    }
+
+    public ClassName getClassName() { return name; }
+
+    public static int[] getHealthValues(ClassName name) {
+        int[] vals = null;
+
+        switch(name) {
+            case Brute:
+            case Cragheart:
+                vals = new int[]{0, 10, 12, 14, 16, 18, 20, 22, 24, 26};
```

Comment:
personally, i am not a big fan of having static data within functions - i feel like this entire method could be a map (or at least do the look up from a map) as an instance variable.  it would be easier to modify later too as the data is defined in one place rather than in a switch statement.  maybe not worth the effort if you're planning on pulling these values from a db later though.

Reply:
I gotcha. the `put` thing is definitely annoying but I should be able to do the math on finding teh amount of exp for needed for each level so I could at least `for` it